api/ingress/v1alpha1/domain_types.go

```diff
@@ -112,26 +152,6 @@ func init() {
 	SchemeBuilder.Register(&Domain{}, &DomainList{})
 }
 
-// SetStatus pulls the fields off the ngrok domain and sets each one on the status field of the domain
-func (d *Domain) SetStatus(ngrokDomain *ngrok.ReservedDomain) {
-	d.Status.ID = ngrokDomain.ID
-	d.Status.Region = ngrokDomain.Region
-	d.Status.Domain = ngrokDomain.Domain
-	d.Status.URI = ngrokDomain.URI
-	d.Status.CNAMETarget = ngrokDomain.CNAMETarget
-}
-
-// Equal returns true if the domain status is equal to the ngrok domain
-func (d *Domain) Equal(ngrokDomain *ngrok.ReservedDomain) bool {
-	return d.Status.ID == ngrokDomain.ID &&
-		d.Status.Region == ngrokDomain.Region &&
-		d.Status.Domain == ngrokDomain.Domain &&
-		d.Status.URI == ngrokDomain.URI &&
-		ptr.Equal(d.Status.CNAMETarget, ngrokDomain.CNAMETarget) &&
-		d.Spec.Description == ngrokDomain.Description &&
-		d.Spec.Metadata == ngrokDomain.Metadata
```

Comment:
This function felt a bit strange and was only used in 1 spot. It seemed also like this was the only file which had `ngrok-api-go` included to do this so its not a pattern others have